task_processing/plugins/smartstack.py

```diff
@@ -0,0 +1,34 @@
+import yaml
+
+
+YOCALHOST_IP = '169.254.255.254'
+TASKPROC_SERVICE_FILE = '/nail/etc/services/task_processing/smartstack.yaml'
```

Comment:
right, ip is just as useless to anybody outside yelp as the service file, which is exactly why i don't think it belongs in this repo. the fewer magic values we have hardcoded here - the better, lets make users provide all the inputs.